include/kwk/utility/container/shape.hpp

```diff
@@ -376,9 +406,21 @@ namespace kwk
     //! @return   An instance of @ref kwk::shape corresponding to the shape of the selected
     //!           sub-volume
     //==============================================================================================
+    // has to be defined inline due to (Apple) Clang deficiencies
+    // https://github.com/llvm/llvm-project/issues/58952
+    // but also after of_size
+    //==============================================================================================
     template<typename... Slicers>
-    inline constexpr auto operator()(Slicers const&... s) const noexcept
-    requires( sizeof...(Slicers) <= static_order);
+    constexpr auto operator()(Slicers const&... s ) const noexcept
```

Comment:
where is the inline then ?

Reply:
inline as in within the class (check the llvm issue/godbolt link) ... the inline is superfluous here (for templates and in-class-definition defines)

Reply:
Oh OK I misunderstood. Fine then